ci/docker-entrypoint.sh

```diff
@@ -13,4 +13,12 @@ if [ "$1" = 'envoy' ]; then
 	fi
 fi
 
-exec "$@"
+if [ -n "$ENVOY_UID" ]; then
```

Comment:
What if a user want to run Envoy as root? Non-root user cannot listen on well-known ports. Setting ENVOY_UID to 0 will fail at usermod.

Reply:
this has been updated, so if `ENVOY_UID=0` it follows the existing `exec` codepath and avoids `usermod`, `su-exec` etc